rpm/debugedit

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "Warning: legacy /usr/lib/rpm/debugedit invoked" >&2
```

Comment:
We don't need this script just symlink debugedit into the rpm directory in the debugedit package.

Reply:
I opted to this approach so we can easily see where the legacy path is still used and update where necessary.

Reply:
Hm? That isn't the legacy path. Calling debugedit as /usr/lib/rpm/debugedit is just fine. Besides debugedit isn't invoked by anyone but RPM.

Did you found anyone else calling debugedit?

Reply:
[Fedora](https://src.fedoraproject.org/rpms/debugedit/blob/cb092fd6d27ca5ee0857051e77e914cb2490b89f/f/debugedit.spec#_82) installs the files in /usr/bin but [openSUSE](https://build.opensuse.org/projects/openSUSE:Factory/packages/debugedit/files/debugedit.spec?expand=1) installs them in /usr/lib/rpm, so that's debatable. Moving them in /usr/bin makes more sense to me, and that is the simplest solution to provide the binaries and scripts while updating RPM past 4.16 later.

I didn't really pick out any binaries but created compatibility scripts for all three files that are moved. Most simple way and works for all.